src/conf/tasks/noisy_kalman_filter.yaml

```diff
@@ -0,0 +1,17 @@
+inherit: 
+    - ../wandb.yaml
+
+training:
+    task: noisy_kalman_filter
+    task_kwargs: {"hidden_layer_size": 100}
```

Comment:
Do we want to parametrize what distribution we sample "noise" from?

Reply:
honestly, we could, but I sort of thought since parameter matrices are more are less all "similar" in terms of eigenvalues, if I just fixed the noise, it would have a similar impact on all of the Kalman filter systems. But if we want, we could look into increasing noise as a curriculum parameter.